x-pack/legacy/plugins/reporting/server/routes/generate_from_savedobject_immediate.ts

```diff
@@ -44,9 +44,7 @@ export function registerGenerateCsvFromSavedObjectImmediate(
       const logger = parentLogger.clone(['savedobject-csv']);
       const jobParams = getJobParamsFromRequest(request, { isImmediate: true });
       const createJobFn = createJobFactory(server);
-      // By passing the request, we signal this as an "immediate" job.
-      // TODO: use a different executeFn for immediate, with a different call signature, to clean up messy types
-      const executeJobFn = executeJobFactory(server, request);
```

Comment:
this comment was just wrong, and `executeJobFactory` is a memoized function that can't use the 2nd param of the function call anyway.